### DISCUSSION TITLE
Using type hints

### DISCUSSION THREAD

**Discussion by kangz12345** [snu-quiqcl/qiwis#75] [Ideas]
As you can see in [PEP 484 - Type Hints](https://peps.python.org/pep-0484), type hints can improve static analysis of the codes.
Moreover, I think the code would be clearer and less documentation will be required.

How about forcing to use type hints in our repo?

**Comment by BECATRUE** [comment]
It looks good!

We already use type hint a little, so I think it isn't a bothering work.

**Comment by Ahrrri** [comment]
It looks good to me too!
